src/platform/user/profile/vet360/components/AddressField/AddressEditModal.jsx

```diff
@@ -92,13 +92,16 @@ class AddressEditModal extends React.Component {
   };
 
   /**
-   * Helper function that:
-   * - totally removes data fields that are not set
-   * - sets the form data's `view:livesOnMilitaryBase` prop to `true` if this is
+   * Helper function that calls other helpers to:
+   * - totally remove data fields that are not set
+   * - set the form data's `view:livesOnMilitaryBase` prop to `true` if this is
    *   an overseas military mailing address
+   *
+   * If the argument is not an object this function will simply return whatever
+   * was passed to it.
    */
   transformInitialFormValues = initialFormValues => {
-    if (!initialFormValues) {
+    if (!(initialFormValues instanceof Object)) {
```

Comment:
This change isn't needed to fix this particular bug. It's a follow up change on my last PR.

Reply:
`instanceof` always seems a little spooky to me. Sometimes it seems like we could have had a smoother pattern in place to not have to check the argument type. How come it's necessary here?

Reply:
It's not entirely necessary. I made this change just to make the code a little more specific about what it was testing for.